src/components/organisms/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, SVGProps } from 'react'
+import BaseButton from 'components/atoms/BaseButton/BaseButton'
+import { showModal, ModalTypes } from 'components/organisms/modals/ModalRoot'
+import { IconProps } from 'components/molecules/Button/Button'
+
+interface TooltipProps {
+  icon?: FC<SVGProps<SVGSVGElement>>
```

Comment:
**[blocker]**

Let's add the default icon here. In most cases we will use the questionmark icon for this, so let's use that as the default.